src/dicts.jl

```diff
@@ -126,14 +126,14 @@ end
 
 function subset(x::DictRowTable, inds; viewhint::Union{Bool,Nothing}=nothing, view::Union{Bool,Nothing}=nothing)
     if view !== nothing
-        @warn "`view` keyword argument is deprecated for `Tables.subset`, use `viewhint` instead"
+        @warn "`view` keyword argument is deprecated for `Tables.subset`, use `viewhint` instead" maxlog=10
```

Comment:
unrelated change but deprecation-style warnings should generally get a `maxlog` imo